src/Line.js

```diff
@@ -410,6 +435,43 @@ export function Line(opts, data, then) {
 	self.ctx = ctx;
 
 	plot.appendChild(can);
+	
+	self.resize = updateChartDimensions;
+
+	function resize(newWidth, newHeight) {
+		updateChartDimensions(newWidth, newHeight);
+		Object.keys(scales).forEach(id => setScale(id, scales[id].min, scales[id].max));
```

Comment:
the rest of the codebase uses `for-in` iteration over `scales` to avoid array allocation that `Object.keys` does